src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -100,7 +101,7 @@ public static CommitResult analyzeCommit(CommitInfo commitInfo, RepoConfiguratio
      */
     private static void extractTagNames(String[] tags) {
         for (int i = 0; i < tags.length; i++) {
-            tags[i] = tags[i].substring(tags[i].lastIndexOf("tag: ") + 5);
+            tags[i] = tags[i].substring(tags[i].lastIndexOf(TAG_TITLE) + 5);
```

Comment:
On this line, you have purposely added a space after `tag:` isn't it? hence the constant `5` over there.
To make it "safer" usage, probably use `TAG_TITLE.length()` instead of 5, and `.trim()` the result or just use `tag: ` in the first place

Reply:
yep noted. thanks for the advice.